### ISSUE TITLE
[Bug Report] eye icon on textfield does not work on enter 

### ISSUE THREAD

**Issue by rveruna** [vuetifyjs/vuetify#10398]
### Environment
**Vuetify Version:** 2.2.8
**Vue Version:** 2.6.11
**Browsers:** Chrome 79.0.3945.130
**OS:** Windows 10

### Steps to reproduce
tab through password field using tabulator
stop at icon
press enter

### Expected Behavior
it should show/hide password

### Actual Behavior
does nothing, only working on click

### Reproduction Link
<a href="https://codepen.io/rveruna/pen/rNaEJzZ" target="_blank">https://codepen.io/rveruna/pen/rNaEJzZ</a>

### Other comments
you can see that on your documentation page as well

**Comment by denozavr** [comment]
I guess we can say the same about `clearable`, this icon focused on `Tab` key, but when the `Enter` is pressed then nothing happens, could be easily reproduced/seen from docs demo https://vuetifyjs.com/en/components/text-fields#clearable

**Comment by rveruna** [comment]
Thank you
